config/rollup.config.js

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+
+import makePlugins from './plugins/index';
+import makeDevServerPlugins from './plugins/server';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+// default config that is used across all builds
+const config = {
+  input: 'index.js',
+  treeshake: {
+    // We assume reading a property of an object never has side-effects so we
+    // disable this option as it can significantly reduce bundle size but
+    // can potentially break functionality if you rely on getters or errors
+    // from illegal property access.
+    propertyReadSideEffects: false
+  }
+};
+
+export default function makeConfig(commandOptions) {
+  const builds = [];
+
+  // iife build (for use with dev-server)
+  const iife = {
+    ...config,
+    plugins: [
+      ...makePlugins(isProduction),
+      // if rollup has been ran with `--open`, include the dev server plugins
+      ...(commandOptions.open ? makeDevServerPlugins(commandOptions) : [])
+    ],
+    output: {
+      format: 'iife',
+      name: 'Spectacle',
+      file: path.join('dist', 'spectacle.js')
```

Comment:
Can we name this `dist/example.js` or something so that it's very obvious what this is vs. the real library?